dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.jaxb-3.0.feature

```diff
@@ -9,7 +9,8 @@ IBM-Process-Types: client, \
 Subsystem-Name: Jakarta XML Bindings 3.0
 -features=\
   com.ibm.websphere.appserver.classloading-1.0, \
-  io.openliberty.jakarta.activation-2.0
+  io.openliberty.jakarta.activation-2.0, \
```

Comment:
> Should we make a jaxb-2.2 API feature with eeCompatible in it and have features depend on it instead of pulling in the bundle directly?

Probably, will create a new issue to gather up the requirements for these new private features.